rllib/core/models/catalog.py

```diff
@@ -13,9 +16,100 @@
     CNNEncoderConfig,
 )
 from ray.rllib.models import MODEL_DEFAULTS
+from ray.rllib.models.distributions import Distribution
 from ray.rllib.models.utils import get_filter_config
 from ray.rllib.utils.error import UnsupportedSpaceException
 from ray.rllib.utils.spaces.simplex import Simplex
+from ray.rllib.utils.spaces.space_utils import flatten_space
+
+
+def _multi_action_dist_partial_helper(
```

Comment:
This is a helper method to create the `MultiActionDistribution` partials.
We use this for tests and the `Catalog. get_dist_cls_from_action_space()` method.
If a user wishes to create distributions that exceed this method of the catalog, they should overwrite it entirely to represent their use-case and don't have to rely on this wonky method.